src/test/eo/org/eolang/math/number-tests.eo

```diff
@@ -1064,11 +1064,42 @@
       number e
     $.equal-to 1.0
 
-[] > ln-of-float-ten
+[] > ln-taylor-check
   assert-that > @
-    ln.
-      number 10.0
-    $.equal-to 2.302585092994046
+    number.ln.ln-if-greater-than-1.term-2.ln-taylor
+      0.5
+      50
+    $.equal-to 0.4054651081081644
```

Comment:
@levBagryansky why we check the equality and not that value placed between 2 related values?

Reply:
@EugeneDar please check https://github.com/objectionary/eo-math/blob/master/src/test/eo/org/eolang/math/number-tests.eo#L649. Seems it is ok because the result is determined

Reply:
@levBagryansky I don't think we should be tied to a specific implementation